etna/transforms/outliers/point_outliers.py

```diff
@@ -57,7 +57,9 @@ def detect_outliers(self, ts: TSDataset) -> Dict[str, List[pd.Timestamp]]:
         :
             dict of outliers in format {segment: [outliers_timestamps]}
         """
-        return get_anomalies_median(ts=ts, in_column=self.in_column, window_size=self.window_size, alpha=self.alpha)
+        return get_anomalies_median(
+            ts=ts, in_column=str(self.in_column), window_size=self.window_size, alpha=self.alpha
```

Comment:
Why do we need  applying `str` here?

Reply:
In the base class we have typing `Union[Literal["all"], List[str], str]`, so mypy raise warning

Reply:
I don't think that casting to `str` is a good solution because almost anything can be casted to str (even if `self.in_column` is list of "all"). We can
1. ignore typing here
2. somehow define that in this class `self.in_column` is str.

Reply:
Now the base class has attribute `required_features` not `in_column`, this should fix the possible problems